const.ts

```diff
@@ -0,0 +1,76 @@
+export const COMMANDS: {
```

Comment:
I prefer having this sort of stuff separate. You'll  see it cleans up the `main.ts` file a bit